### ISSUE TITLE
Define columns automatically

### ISSUE THREAD

**Issue by KieranP** [kristianmandrup/acts_as_list_mongoid#1]
It would be great if all that was needed was

```
include Mongoid::ActsAsList
```

Then it'd automatically create a position field, and initialize the code without having to call `field` yourself or calling `acts_as_list`.

Is this possible?

**Comment by kristianmandrup** [comment]
Good idea! I will look into it. It could even be made possible to state it like this.

class Whatever
  acts_as_list :mongoid
end

or

class Whatever
  Mongoid.acts_as_list
end

Which do you think is better?

**Comment by KieranP** [comment]
Given the convention of mongoid things is using include, a call to acts_as_list shouldn't be needed.

An example:

```
class Post
  include Mongoid::Document
  include Mongoid::Timestamps
  include Mongoid::Versioning
  include Mongoid::ActsAsList
end
```

Then Mongoid::ActsAsList would do something like:

```
module Mongoid
  module ActsAsList
    included do |klass|
      klass.field :position
      klass.acts_as_list
    end
  end
end
```

**Comment by kristianmandrup** [comment]
ah ok ;) I've just started to use class level macros recently as it feels more DSL like and using include is much more "low-level". But I guess if the convention is not to use DSL then let's stick to that for now. It might also be a case of too much abstraction.
